tests/x11/reboot_and_install.pm

```diff
@@ -15,14 +15,20 @@ use strict;
 
 use testapi;
 use utils qw(power_action workaround_type_encrypted_passphrase);
-use version_utils 'is_sle12_hdd_in_upgrade';
+use version_utils qw(is_sle12_hdd_in_upgrade sle_version_at_least);
 
 use bootloader_setup;
 use registration;
 
 sub run {
     # reboot from previously booted hdd to do pre check or change e.g. before upgrade
-    power_action('reboot');
+    if (sle_version_at_least('12')) {
+        power_action('reboot');
+    }
+    else {
+        select_console 'root-console';
+        type_string "reboot\n";
+    }
```

Comment:
I have a little concern here: the change above makes "x11/reboot_and_install.pm" not really act as documented "Summary: Ensure the system can reboot from gnome".

Reply:
The reboot behavior of SLE11 is quite different with SLE12, the fuction power_action('reboot') can not work with SLES11. It would be a workaround for SLES11 here, I would add a comment to it.

Reply:
Yes, it will be more flexible. Open a new ticket to track it.
https://progress.opensuse.org/issues/30532

Reply:
Is it possible to update power_action within this ticket?